src/index.html

```diff
@@ -5,9 +5,48 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./style.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div data-qa="card" class="card">
+      <div class="card__icon">
+        <img src="../src/images/imac.jpeg" alt="apple imac">
+      </div>
+      <div class="card__title">
+        <div class="card__text">
+          <span>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</span>
+        </div>
```

Comment:
```suggestion
        <h2 class="card__title">
          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
        </h2>
```
Product code is not part of the title
use semantic tag for title